inc/ProgEvent.h

```diff
@@ -90,9 +74,9 @@ namespace PCOE {
 
         /** @brief Set the number of timestamps for which prognostic relevant prognostic data
```

Comment:
"prognostic relevant prognostic data"? I think we can remove at least one instance of the word "prognostic" here...